src/config.ts

```diff
@@ -240,3 +241,17 @@ export const manageFlowEnableUserList = withDefault(
       `Invalid Manage Flow enable user list configured: ${process.env.MANAGE_FLOW_ENABLE_USER_LIST}`
     );
   });
+
+/**
+ * Lock the creation of a new APIM user, when resolve SelfCareIdentity.
+ */
+export const lockSelfcareCreateNewApimUser = withDefault(
+  BooleanFromString,
+  ("false" as unknown) as boolean
+)
+  .decode(process.env.LOCK_SELFCARE_CREATE_NEW_APIM_USER)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** Is this a Feature Flag? (that is: you meant this flag to enable/disable the feature after deploy). Consider add the prefix `FF_` to the env variable (not mandatory)‏